app/src/main/java/org/literacyapp/authentication/TrainingHelper.java

```diff
@@ -301,4 +330,30 @@ private File createModelDownloadFile(File modelFile){
         }
         return modelDownloadFile;
     }
+
+    /**
+     * Create the Avatar file for a Student using a StudentImage
+     * @param studentImage
+     * @return
+     */
+    private File createAvatarFileFromStudentImage(StudentImage studentImage){
+        String dateFormatted = (String) DateFormat.format("yyyy-MM-dd_HHmmss", Calendar.getInstance());
+        String imageFilePath = StudentHelper.getStudentAvatarDirectory() + "/" + DeviceInfoHelper.getDeviceId(context) + "_" + dateFormatted + ".png";
+        File avatarFile = new File(imageFilePath);
+        try {
+            Log.i(getClass().getName(), "createAvatarFileFromStudentImage: Preparing InputStream and OutputStream to copy the StudentImage into the Avatar directory");
+            Log.i(getClass().getName(), "createAvatarFileFromStudentImage: InputStream with the file: " + studentImage.getImageFileUrl());
+            InputStream inputStream = new FileInputStream(studentImage.getImageFileUrl());
```

Comment:
A StudentImage is 224 x 224. I guess then I need to resize it to 96 x 96 to use as an avatar?